scripts/monorepo/getChangedPackages.js

```diff
@@ -0,0 +1,27 @@
+const spawnSync = require('child_process').spawnSync;
+const findGitRoot = require('./findGitRoot');
+
+/**
+ * @param {string} since - Commit to compare against
+ * @returns {Set<string>} - Set of packages that are changed
+ */
+function getChangedPackages(since = 'origin/master') {
```

Comment:
This already exists in `workspace-tools`, so you should probably just import it from there instead of re-implementing.

Reply:
The `workspace-tools` one is different, I'm using Lage here to get the changed packages and their dependents... maybe I should rename this ?

Reply:
I'll rename this to `getAffectedPacakges`